### ISSUE TITLE
Debian Jessie ships too old Node version

### ISSUE THREAD

**Issue by Ryuno-Ki** [pattern-lab/patternlab-node#236]
Today I tried to install patternlab-node on a Debian Jessie (aka v8) with nodejs installed [from repos](https://packages.debian.org/de/jessie/nodejs) (that is, in v0.10.29).

After executing `grunt serve` I received the following message:
```bash
$ grunt serve
Running "patternlab" task

Running "copy:main" (copy) task
Copied 28 files

Running "browserSync:dev" (browserSync) task

/home/aj/src/patternlab/node_modules/patternlab-node/node_modules/bs-html-injector/node_modules/jsdom/lib/jsdom.js:3
`jsdom 4.x onward only works on io.js or Node.js 4 or newer: https://github.co
^
Fatal error: Unexpected token ILLEGAL
```

Downgrading to patternlab 0.10.1 "solved" the issue. This is just for your information. If you have spare time, you may want to backport some features.

As far as I can tell, the only issue is some UTF-8 error on grunt browser sync, but that's okay. I refresh the browser manually then.

**Comment by bmuenzenmeyer** [comment]
Morning @Ryuno-Ki 

I received similar errors as the node community (core and all dependencies) moved to 4.X and 5.X. According to https://nodejs.org/en/download/releases/ version 0.10.29 is about a year and a half old, are newer versions of node available on Debian Jessie? I've never done a Linux installation, but more info can surely be found here: https://nodejs.org/en/download/

All that said, version 0.10.1 of patternlab node is still a quite-capable release, and as long as you are okay staying locked you can still build some nice things with it.

**Comment by Ryuno-Ki** [comment]
As far as I [could see](https://nodesource.com/blog/nodejs-v012-iojs-and-the-nodesource-linux-repositories/) there were the option to get at least Node v0.12, but that seem to require building from source. Especially the installation through "sudo" (that is, systemwide) could causes conflicts with existing projects.

I would rather opt for trying [nvm](/creationix/nvm) or [n](/tj/n).

**Comment by bmuenzenmeyer** [comment]
ah yes that could be a hassle, but I have had success with nvm in the past.

I am going to close this issue as unrelated to patternlab-node

**Comment by greenc** [comment]
Apologies for revisiting a closed issue, but I stumbled upon it today while trying to get `grunt serve` to work using patternlab-node v.1.2.0 on node v.0.12.5.

The workaround I implemented was to specify an older version of the bs-html-injector dependency in the patternlab-node package.json file. Specifically, changing `"bs-html-injector": "^3.0.0"` to `"bs-html-injector": "^2.0.0"`.

Not an ideal solution, but good enough for my purposes, and maybe for others who run into the same issue.
